### PR TITLE
Medyanet Ad Network Update for Domain Data 

### PR DESCRIPTION
**Pull request by mozdenn**
Added new custom ad-tag data parameter: Domain

### CODE REVIEW

**Comment by calebcordry** [comment]
Hello @mozdenn are you still working on this? If so, could you please follow the guidelines here: https://github.com/ampproject/amphtml/blob/master/ads/README.md

Thanks!

File: ads/medyanet.js [hunk 1]
```diff
@@ -23,10 +23,11 @@ import {validateData} from '../3p/3p';
  */
 
 export function medyanet(global, data) {
-  validateData(data, ['slot']);
+  validateData(data, ['slot', 'domain']);
```
Comment (calebcordry) [inline ads/medyanet.js]: Do you have any optional arguments that you would like to validate here?

Comment (mozdenn) [reply ads/medyanet.js]: yes, slot and domain parameters are required.

Comment (calebcordry) [reply ads/medyanet.js]: Sorry if I wasn't clear. I understand that these are the required arguments. You can also add optional parameters as a third argument if you wish. It is optional, so if you are ready I will go ahead and merge.

**Pull request merged**